test/any/web_apis/web-apis-integration.es6.js

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
nit: here and other test files: file name: `web_apis-integration.es6.js` to match `<file-under-test>-integration.es6.js`.